client/core/src/com/cows/game/roundSimulation/simulationModels/UnitSimulationModel.kt

```diff
@@ -0,0 +1,41 @@
+package com.cows.game.roundSimulation.simulationModels
+
+
+class UnitSimulationModel(
```

Comment:
Some of these values, namely movementProgress, isSpawned and isDead don't need to be set in the constructor, as they should always have 0, false, false respectively as initial values